projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/sentence/entity/SentenceRepository.kt

```diff
@@ -3,6 +3,7 @@ package uk.gov.justice.digital.hmpps.integrations.delius.sentence.entity
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.data.jpa.repository.Query
 import uk.gov.justice.digital.hmpps.integrations.delius.overview.entity.Event
+import java.time.LocalDate
```

Comment:
Is this used?

Reply:
If you're committing via IntelliJ, I'd recommend ticking the "Optimise imports" and "Re-format code" options in the commit dialog so it sorts these out for you automatically.

Reply:
done